docs/developer-docs/latest/developer-resources/content-api/content-api.md

```diff
@@ -489,6 +489,8 @@ GET http://localhost:1337/restaurants/count
 
 Creates an entry and returns its value.
 
+If the [Internationalization (i18n) plugin](/developer-docs/latest/development/plugins/i18n.md) is installed, it's possible to use POST requests to the Content API to [create localized entries](/developer-docs/latest/development/plugins/i18n.md#creating-a-new-localized-entry).
```

Comment:
I wonder if it'll scale when we'll have more plugin that wants to add there specificity

Reply:
Great point! And it probably won't scale.
When other plugins will be released, I plan on creating a new doc in Strapi plugins to describe the impact of plugins on core features, something like [what Mégane did for the user guide](https://github.com/strapi/documentation/pull/223/files#diff-f915e8e847782b68b63a47f132c6fe39d224af7eb09c03aa651a73fa8daf0780R23).
At this stage, I think it would be overkill for dev docs — but open to discussing about it :-)